themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1792,6 +1791,12 @@ html body.tc-body.tc-single-tiddler-window {
 	width: 100%;
 }
 
+.tc-controlpanel svg.tc-image-button{
```

Comment:
@pmario, I will need to investigate what it was before. All in all, this part does not need to / should not be changed by this PR. It's an artifact.